db/seeds.rb

```diff
@@ -549,6 +549,51 @@ def require_updates?(real_attrs, seed_attrs)
   is_project: true,
   url: "/web_development_101/project_rails.md"
 )
+# +++++++++++
+# SECTION
+# +++++++++++
+
+section_position += 1
+section = create_or_update_section(
+  title: "Pair Programming",
```

Comment:
here is the line I added to the pairing project lesson:

https://github.com/TheOdinProject/curriculum/pull/5502